src/components/Public/Portal.js

```diff
@@ -15,7 +19,11 @@ const Portal = () => {
         })
         .then(data => {
             const postings = data.allPostings;
-            const visiblePostings = postings.filter(posting => posting["isVisible"] === true);
+            //convert the time
+            var d = new Date();
+            var formattedDate = d.getFullYear() + "-0" + (d.getMonth() + 1)  + "-0" + d.getDate();
+            console.log(formattedDate);
+            const visiblePostings = postings.filter(posting => (posting["isVisible"] === true && dateTime(posting["deadline"]) >= formattedDate));
```

Comment:
I see why this isn't working – it's because you're comparing the string, not the date object. (https://stackabuse.com/compare-two-dates-in-javascript) Try refer to the link and it'll make some sense!

Ideally, you'd want to parse the string of each posting and convert that time into a Date object.

https://stackoverflow.com/questions/5619202/converting-a-string-to-a-date-in-javascript

Let me know if you have any questions!

Reply:
I ended up comparing strings and it worked out. Since the date object (even after converting to datestring), the format is still different so the easiest way I think to compare the two is to convert to same string format and compare the strings.